crates/crypto/core/src/lib.rs

```diff
@@ -10,7 +10,7 @@ pub mod aggregation;
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 pub enum KeyTypeId {
-    #[cfg(feature = "bn254")]
+    #[cfg(any(feature = "bls", feature = "tangle"))]
```

Comment:
This is wrong?

Reply:
Tangle doesn't have bn254 keys, yes. However in the code, if you requested this key, it will return an error to the caller with an Unsupported KeyType error.